_maps/map_files/biodome/modules/map legend.txt

```diff
@@ -0,0 +1,19 @@
+[rooms.arrivalsecupper]
+modules = ["arrivalsecupper_1.dmm", "arrivalsecupper_2.dmm", "arrivalsecupper_3.dmm"]
+# 1. Small S connection that connects Arrivals, EVA and Security. Has a simple side room with misc maintenance gear.
```

Comment:
Did you forget to delete this from Tramstation?

Reply:
Oh whoops. Yes 😔